pkg/scheduler/core/extender_test.go

```diff
@@ -164,15 +172,19 @@ func (f *FakeExtender) selectVictimsOnNodeByExtender(
 	pod *v1.Pod,
 	node *v1.Node,
 	nodeNameToInfo map[string]*schedulercache.NodeInfo,
-) ([]*v1.Pod, int, bool) {
+) ([]*v1.Pod, int, bool, error) {
 	// TODO(harry): add more test in generic_scheduler_test.go to verify this logic.
 	// If a extender support preemption but have no cached node info, let's run filter to make sure
 	// default scheduler's decision still stand with given pod and node.
 	if !f.nodeCacheCapable {
-		if fits, _ := f.runPredicate(pod, node); !fits {
-			return nil, 0, false
+		fits, err := f.runPredicate(pod, node)
+		if err != nil {
+			return nil, 0, false, err
 		}
-		return []*v1.Pod{}, 0, true
+		if !fits {
+			return nil, 0, false, err
```

Comment:
Here the value of err is nil, perhaps we should give an ‘no fits’ error.

Reply:
This is expected to be nil, because for the caller side, `unfit` is different from `error`. But I can change it to `return nil, 0, false, nil` for consistency.